src/utils/__init__.py

```diff
@@ -1,5 +1,5 @@
 from utils.scraping import chrome_browser_setup
 from utils.scraping import get_carbon_intensity
-from utils.cpu_usage import get_cpu_info
-from utils.cpu_usage import get_cpu_usage
+from utils.cpu import get_cpu_info
+from utils.cpu import get_cpu_usage
```

Comment:
Didn't we decide to not use the __init__.py files to import each single function from the modules? Otherwise we would need to add a line every time we add a function.

Reply:
Oh just realised I was the one supposed to clean the init files sorry, gonna do it asap. In the meanwhile maybe we can just avoid using it too much.

Reply:
yep, I left the init file with the same criteria to let you change it